examples/bri-3/src/bri/zeroKnowledgeProof/services/blockchain/ethereum/contracts/Ccsm.sol

```diff
@@ -0,0 +1,16 @@
+//SPDX-License-Identifier: UNLICENSED
+pragma solidity 0.8.17;
+
+contract Ccsm {
+  mapping(string => bool) public anchorHashStore;
+
+  function setAnchorHash(string calldata _anchorHash) external {
```

Comment:
this should be protected by OnlyOwner, so that only the BPI can store a new hash. But let's tackle in detail as part of milestone 5, please open a placehodler issue related to this

Reply:
Done. #751